sandbox/cgroup/scanning_test.go

```diff
@@ -72,7 +72,17 @@ func (s *scanningSuite) TestSecurityTagFromCgroupPath(c *C) {
 	c.Check(cgroup.SecurityTagFromCgroupPath("/a/b/snap.foo.foo.service/"), DeepEquals, mustParseTag("snap.foo.foo"))
 }
 
-func (s *scanningSuite) writePids(c *C, dir string, pids []int) {
+func arrayContainsString(arr []string, needle string) int {
```

Comment:
(nitpick) I would follow `strutil.ListContains()` a bit here, maybe rename to
```
func listContainsN((list []string, str string) int
```
?

Reply:
Changed to "matchesInArray()".